lang/en/facetoface.php

```diff
@@ -410,6 +410,7 @@
 
 You will receive a reminder [reminderperiod] business days before this course.
 ';
+$string['setting:defaultconfirmationmessagedefault2'] = '<p><b>Thank you for signing up!</b>';
```

Comment:
There is an unclosed <p> tag here, that will need to be fixed.
Ideally, there wouldn't be HTML in lang strings. But sometimes it is unavoidable.

Reply:
This is also fixed in issue #74